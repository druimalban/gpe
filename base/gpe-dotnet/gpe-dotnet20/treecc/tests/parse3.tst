// test bad declaration syntax

%abstract %typedef
%node x %typedef
; []
{ xyz }
%node y x %unknown
