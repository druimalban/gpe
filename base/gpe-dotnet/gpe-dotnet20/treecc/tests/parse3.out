parse3.tst:3: declaration expected
line 4: %node x no_parent 4
parse3.tst:5: declaration expected
parse3.tst:7: unknown keyword `%unknown'
line 7: %node y x 0
parse3.tst:7: declaration expected
